src/web/basics/form-elements/text-field/text-field.scss

```diff
@@ -43,14 +40,14 @@
     transition-delay: 0.1s;
 
     .hig__text-field__input--no-value + & {
-        top: 22px;
+        top: -2px;
         left: 1px;
         font-size: 14px;
         line-height: 20px;
     }
 
     .hig__text-field__input--no-value.hig__text-field__input:focus + & {
-        top: 0;
+        top: -24px;
```

Comment:
Generally, this is what causes the label to move up above the field when the field gets focus.

The reason this line changes in this commit is that I changed the markup for input a little, and so needed the label to go up higher.